<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="config_path" default="./config.json" />
  <arg name="input_topic" default="camera/image_raw" />
  <node name="yolo_seg_node" pkg="yolo_seg" exec="yolo_seg_node"> <!--  respawn="true" -->
    <param name="config_path" value="$(var config_path)"/>
    <param name="input_topic" value="$(var input_topic)"/>
  </node>
</launch>
